### PR TITLE
Use vaccines from metrics

### PR DESCRIPTION
**Pull request by ghop02**
Update the data snapshot, api types, and code to use metrics for api values

### CODE REVIEW

File: src/common/models/Projection.ts [hunk 1]
```diff
@@ -364,36 +369,47 @@ export class Projection {
...
 
-    return null;
+    const peopleVaccinated = actuals.vaccinationsCompleted;
+    const peopleInitiated = actuals.vaccinationsInitiated;
+    const percentInitiated = metrics.vaccinationsInitiatedRatio;
+    const percentVaccinated = metrics.vaccinationsCompletedRatio;
+
+    if (
+      peopleVaccinated === null ||
+      peopleVaccinated === undefined ||
```
Comment (ghop02) [inline src/common/models/Projection.ts:387]: There's probably a better way to do this, suggestions included :)

Comment (pnavarrc) [reply src/common/models/Projection.ts:387]: I would use `Number.isFinite` instead, it returns `false` for: `NaN`, `-Infinity` and `+Infinity`, `null` and `undefined`

**Review by pnavarrc — APPROVED** [review]
Looks good! Added a suggestion for the validation

File: src/common/models/Projection.ts (hunk 1 above)
Comment (ghop02) [reply src/common/models/Projection.ts:387]: ah this doesn't quite work with typescript.  will attempt to do later

**Pull request merged**